tailwind.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
         hireTalent: "#FDC0E33D",
         "event-bg": "#210D15",
         tutu: "#F9E8F1",
+        "custom-grey-3": "#434343",
```

Comment:
use this website to get actual color names, these don't look good [color-blindness](https://www.color-blindness.com/color-name-hue/ )